oak_remote_attestation/src/dice.rs

```diff
@@ -191,32 +191,8 @@ impl TryFrom<DiceData> for DiceBuilder {
 impl TryFrom<Stage0DiceData> for DiceData {
     type Error = anyhow::Error;
     fn try_from(value: Stage0DiceData) -> anyhow::Result<Self> {
-        let (platform, remote_attestation_report) =
```

Comment:
This change will conflict with #4487. I will resolve the conflictg once #4487 is merged.